docs/config/index.md

```diff
@@ -29,9 +29,11 @@ Since Vite ships with TypeScript typings, you can leverage your IDE's intellisen
 
 ```js
 /** @type {import('vite').UserConfig} */
-export default {
+const config = {
```

Comment:
IntelliSense works fine for me in VS Code with the original code. What IDE are you using?

Reply:
Same here. I changed this at #8590 as it should work now.

Reply:
FYI https://github.com/microsoft/TypeScript/issues/45233 landed in TypeScript 4.5.2 (2021/11).

Reply:
@tony19 I am using VS Code. I updated my TypeScript version and the original code has type now.

Reply:
Let's close the PR then, thanks for sending it anyways @ChocolateLoverRaj. Good to check that the example is correct.